polaris-react/src/components/AppProvider/AppProvider.scss

```diff
@@ -10,8 +10,12 @@
 
 html,
 body {
-  @include text-style-body;
   @include text-emphasis-normal;
+  font-size: 16px;
```

Comment:
Later on in the [file](https://github.com/Shopify/polaris/blob/ed29689180023ae8f9a3eb0316ba9853899a0926/polaris-react/src/components/AppProvider/AppProvider.scss#L40), there's other styling on the body that needed to be hard coded as well because the file doesn't have access to the tokens. So I had to hardcode these values unfortunately.

Reply:
This is a bit confusing why the app provider doesn't have access to the tokens. Previously this made sense but we recently shipped a change to add the tokens globally without the AppProvider injecting them. cc @aaronccasanova @samrose3